LastInfoSec/docker-compose.yml

```diff
@@ -1,15 +1,15 @@
-version: '3'
+version: '3.2'
 services:
-  connector-lastinfosec-cti:
-    image: connector-lastinfosec-cti:0.0.5
+  connector-lastinfosec:
+    image: connector-lastinfosec:0.0.5
```

Comment:
This should be:
`    image: connector-lastinfosec:3.2.0`

Reply:
fixed :)